### DISCUSSION TITLE
Trouble installing raccoon/moose

### DISCUSSION THREAD

**Discussion by ihernandezimbert** [idaholab/moose#25614] [Q&A Getting Started] [answered]
Hi everyone, 

I am getting multiple errors once reaching the make step in moose for installing raccoon. This is what the first few look like and I am not sure how to fix/work around this.

```
/Users/iranhernandezimbert/projects/raccoon/moose/framework/src/utils/PetscDMMoose.C:420:22: error: format specifies type 'long long' but the argument has type 'std::vector<std::pair<std::string, std::string>>::size_type' (aka 'unsigned long') [-Werror,-Wformat]
                     contacts.size(),
                     ^~~~~~~~~~~~~~~~
/Users/iranhernandezimbert/mambaforge3/envs/moose/libmesh/include/libmesh/petsc_macro.h:95:27: note: expanded from macro 'LIBMESH_SETERRQ2'
# define LIBMESH_SETERRQ2 SETERRQ
                          ^
/Users/iranhernandezimbert/mambaforge3/envs/moose/include/petscerror.h:57:127: note: expanded from macro 'SETERRQ'
    PetscErrorCode ierr_seterrq_petsc_ = PetscError(comm, __LINE__, PETSC_FUNCTION_NAME, __FILE__, ierr, PETSC_ERROR_INITIAL, __VA_ARGS__); \
```

Really appreciate the help!

**Comment by GiudGiud** [comment]
Hello

we haven’t seen that one before. Which platform are you installing on?
Can you run the diagnostics script in moose/scripts?

**Comment by cticenhour** [comment, marked as answer]
Racoon is using an older version of MOOSE (3 weeks ago), prior to the recent PETSc update. If you are following our conda instructions, you would be downloading the most recent set of packages, which doesn't work perfectly with that version of MOOSE. 

I recommend that you activate your MOOSE environment in conda and perform the following command:

```
mamba install moose-dev=2023.08.29=build_0
```

This would install a set of packages aligned with the version of MOOSE currently used by Racoon.

**Reply by cticenhour** [reply]
After getting the packages in place, I would then perform a clean of your MOOSE repository:

```
cd /Users/iranhernandezimbert/projects/raccoon/moose
git clean -xfd
```

In order to clean out any old build artifacts that could cause erroneous follow-on issues.

**Reply by ihernandezimbert** [reply]
Thank you - that did it!
